Documentation/GettingStartedWithTheMRTK.md

```diff
@@ -0,0 +1,153 @@
+* Change the Scripting back-end to use IL2CPP (as Unity is deprecating the .NET scripting back-end)
+
+> The only option which is mandatory (and not listed above) is to set Unity to use the .NET 4 scripting runtime by default, as the MRTK uses advanced coding functions only available in .NET4+.  
+> From Unity 2018.3 and above, this will be the default for new Unity Projects.
+
+This prompt will automatically set the above options for you and restart Unity (for the .NET update) ready to start building your MRTK project.
+
+Accepting these options is completely optional (except .NET 4), if you click **Later**, the MRTK will not ask you again and leave you to it.  You can then choose to set these options manually later as you see fit.
+
+### 4. Configure your first Mixed Reality Toolkit scene
+
+The toolkit has been designed so that there is just one object that is mandatory in your scene.  This is there to provide the core configuration and runtime for the Mixed Reality Toolkit (one of the key advantages in the new framework).
+
+Configuring your scene is extremely simple by simply selecting the following from the Editor menu:
+> Mixed Reality Toolkit -> Configure
+
+![](/External/ReadMeImages/MRTK_ConfigureScene.png)
+
+Once this completes, you will see the following in your Scene hierarchy:
+
+![](/External/ReadMeImages/MRTK_SceneSetup.png)
+
+Which contains the following:
+
+* Mixed Reality Toolkit - The toolkit itself, providing the central configuration and activation point for the entire framework.
+* MixedRealityPlayspace - The parent object for the headset, which ensures the headset / controllers and other required systems are managed correctly in the scene.
+* The Main Camera is moved as a child to the Playspace - Which allows the playspace to manage the camera in conjunction with the SDK's
+* UIRaycastCamera added as a child to the Main Camera - To enable seamless UI interactions through the toolkit
+
+> **Note** While working in your scene, **DON'T move the Main Camera** (or the playspace) from the scene origin (0,0,0).  This is controlled by the MRTK and the active SDK.
+> If you need to move the players start point, then **move the scene content and NOT the camera**!
+
+### 5. Hit play
+
+You are now ready to start building your Mixed Reality Solution, just start adding content and get building.
+Switch to other platforms (ensure they have XR enabled in their player settings) and your project will still run as expected without change.
+
+## Configuring your project
+
+The Mixed Reality Toolkit configuration is all centralized on one place and attached to the MixedRealityToolkit object in your active scene.
+
+![](/External/HowTo/MixedRealityToolkitConfigurationProfileScreens/MRTK_ActiveConfiguration.png)
+
+Clicking on this profile will show the configuration screens for the Mixed Reality Toolkit:
+
+![](/External/HowTo/MixedRealityToolkitConfigurationProfileScreens/MRTK_MixedRealityToolkitConfigurationScreen.png)
```

Comment:
Maybe take a screen shot of it locked. We don't want to give people the impression it's broken cause it's grayed out.

Reply:
Easier to talk about it, as it doesn't good locked in a screenshot :D